BookingsApi/BookingsApi/Controllers/JudiciaryPersonController.cs

```diff
@@ -0,0 +1,88 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Threading.Tasks;
+using BookingsApi.Contract.Requests;
+using BookingsApi.DAL.Commands;
+using BookingsApi.DAL.Commands.Core;
+using BookingsApi.DAL.Queries;
+using BookingsApi.DAL.Queries.Core;
+using BookingsApi.Domain;
+using BookingsApi.Validations;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Logging;
+using NSwag.Annotations;
+
+namespace BookingsApi.Controllers
+{
+    [Produces("application/json")]
+    [Route("judiciaryperson")]
+    [ApiController]
+    public class JudiciaryPersonController : Controller
+    {
+        private const string BulkItemErrorMessage = "Could not add or update external Judiciary user with External Id: {0}";
+        private readonly IQueryHandler _queryHandler;
+        private readonly ICommandHandler _commandHandler;
+        private readonly ILogger<JudiciaryPersonController> _logger;
+        
+        public JudiciaryPersonController(IQueryHandler queryHandler, ICommandHandler commandHandler, ILogger<JudiciaryPersonController> logger)
+        {
+            _queryHandler = queryHandler;
+            _commandHandler = commandHandler;
+            _logger = logger;
+        }
+        
+        [HttpPost]
+        [OpenApiOperation("BulkJudiciaryPersons")]
+        [ProducesResponseType((int)HttpStatusCode.OK)]
+        [ProducesResponseType((int)HttpStatusCode.BadRequest)]
+        [ProducesResponseType(typeof(BulkJudiciaryPersonResponse), (int)HttpStatusCode.OK)]
+        public async Task<IActionResult> BulkJudiciaryPersonsAsync(IEnumerable<JudiciaryPersonRequest> request)
+        {
+            var bulkResponse = new BulkJudiciaryPersonResponse();
```

Comment:
no logging middleware in bookings api yet.. could we add info log to the start saying its picking up the request and how many persons are in it?

Reply:
Added it.. but there is also more logging in the Client specifying what page number is being called..etc